src/components/Volunteers/CardVolunteer.jsx

```diff
@@ -408,8 +408,8 @@ export class CardVolunteer extends Component {
             </ListItemAvatar>
             <ListItemText
               primary={`${volunteer.first_name} ${volunteer.last_name || ""}`}
-              secondary={`${volunteer.address.address1 || ""} ${
-                volunteer.address.city || ""
+              secondary={`${volunteer.address.address1 + ""} ${
```

Comment:
is this meant to address the TypeError from the undefined address? Because this is still reading properties from volunteer.address, so it will still fail if that's undefined. Better to use optional chaining on these properties, as well as state and zip below, It also looks like there may be a similar issue around line 442